pkg/testutil/framework.go

```diff
@@ -200,6 +205,16 @@ func (f *Framework) NewTenantAccount(name string, subjects ...rbacv1.Subject) *c
 	}
 }
 
+func (f *Framework) NewFakeDB(name, namespace string) *fakev1alpha1.DB {
```

Comment:
I don't see the big utility in this helper function, rather delete it and create the object the usual way. The name/namespace setting could be generalized with something like this:

```go
WithNameNamespace(obj runtime.Object, name, namespace string) runtime.Object {
    ...
}
```

```go
testutil.WithNameNamespace(&fakev1alpha1.DB{}, "something" , "other") 
```

if you'd like saving code lines per creation from 3-4 to 1.

Reply:
just trying to make it consistent with all other code, for each integration test with have such method, it's easier to extend test later with that method.

Reply:
ok, not my preference but not merge blocking.